docs/guides/operator-guide-azure-enclave.md

```diff
@@ -114,6 +114,12 @@ There are some limitations with regard to location:
 
 - For Azure virtual network deployment availability, check [Linux container groups](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-region-availability#linux-container-groups) in the Azure documentation. You can only deploy to regions with the **Confidential SKU** column set to **Y** in the table.
```

Comment:
@thomasm-ttd FYI I checked this page and I think it's updated, there is no "Confidential SKU" column/tables anymore. Should we consider removing this section? Do we still need to mention `Confidential SKU`?

Reply:
The link has changed to : https://learn.microsoft.com/en-us/azure/container-instances/container-instances-resource-and-quota-limits#confidential-container-resources-preview
Change the text to: 
For Azure virtual network deployment availability, check [Linux container groups](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-resource-and-quota-limits#confidential-container-resources-preview) in the Azure documentation to confirm the availability of Confidential Container regional support.